source/index.html

```diff
@@ -1,12 +1,206 @@
+          <h3 class="product-title section-title">Вязаные корзинки</h3>
+          <p class="product__text">
+            Экологически чистая пряжа, ручная работа. Доступны в шести расцветках и трёх размерах.
+          </p>
+          <table class="product__parameters">
+            <caption class="visually-hidden">Характеристики товара Вязаные корзинки.</caption>
+            <tr>
+              <th>Цвет:</th>
+              <td>«Тиффани»/серый</td>
+            </tr>
+            <tr>
+              <th>Диаметр:</th>
+              <td>25 см</td>
+            </tr>
+            <tr>
+              <th>Высота:</th>
+              <td>15 см</td>
+            </tr>
+          </table>
+          <img  class="product__image" src="img/promo/basket.png" alt="Три вязаных крючком корзинки разных размеров, две серо-зелёных и одна бело-розовая, вкладываются в друг друга по типу матрёшки." width="260" height="315">
+          <p class="product__price">Цена: 990 руб.</p>
+          <button class="button" type="button">Заказать</button>
+        </div>
+      <!-- "Волна" по нижней границе секции является декоративным изображением => через стили -->
+      </section>
+      <section class="advantages">
+        <h2 class="section-title">Коротко о нас:</h2>
+        <!-- Картинки у пунктов списка декоративные, цвет не меняют => через стили -->
+        <ul class="advantages-list">
+          <li class="advantages-list__item advantages-list__item--ecology">Экологически чистые материалы</li>
+          <li class="advantages-list__item advantages-list__item--style">Скандинавский стиль по российской цене</li>
+          <li class="advantages-list__item advantages-list__item--like">Увеличивает лайки на фотографиях</li>
+          <li class="advantages-list__item advantages-list__item--love">Связано вручную с любовью и умилением</li>
+          <li class="advantages-list__item advantages-list__item--domestic">Поддержка отечественного производителя</li>
+          <li class="advantages-list__item advantages-list__item--gift">Поставляется в подарочной упаковке</li>
+        </ul>
+      </section>
+      <section class="reviews">
+        <h2 class="section-title">Отзывы</h2>
+        <ul class="reviews-list">
+          <li class="reviews-list__item">
+            <blockquote class="reviews__content">
+              <p class="reviews__text">
+                Я молодая мама, но всё равно фотографии не набирали и близко такого количества лайков,
+                как у популярных инстамамочек. В отчаянии, я накупила аксессуаров и игрушек в Мишке,
+                и мои фотографии сразу стали более стильными, а также набирают больше лайков!
+              </p>
+              <cite class="reviews__author">
+                Анастасия Красильникова
+                <a class="reviews__link" href="#">@misssssiskras</a>
+              </cite>
+            </blockquote>
+          </li>
+          <li class="visually-hidden reviews-list__item">
```

Comment:
```suggestion
          <li class="hidden reviews-list__item">
```

visually-hidden - вам в лекции не говорили, что это класс только для того, чтобы скрывать от зрячих всякую полезную информацию для незрячих? Это не класс, по которому скрываешь блоки. Скрываешь блоки ты дисплей нан. Для зрячих дисплей нан выше крыши, а вижуали хидден имеет много свойств, чтобы блок скрыть, но при этом скринридер мог его прочесть. Обрати внимание, какая разница между ними.